packages/playwright-test/src/common/fixtures.ts

```diff
@@ -239,7 +239,12 @@ function innerFixtureParameterNames(fn: Function, location: Location, onError: L
     onError({ message: 'First argument must use the object destructuring pattern: '  + firstParam, location });
     return [];
   }
-  const props = splitByComma(firstParam.substring(1, firstParam.length - 1)).map(prop => {
+  const firstParamWithoutBrackets = firstParam.substring(1, firstParam.length - 1);
+  if (firstParamWithoutBrackets.includes('...')) {
```

Comment:
Can we check `props` at the line 251, to be more specific in the error message: `Rest property "...foo" is not supported. List all used fixtures explicitly, separated by comma.`